app/models/facility.rb

```diff
@@ -49,6 +49,11 @@ def accounts
   scope :active, :conditions => { :is_active => true }
   scope :sorted, order: :name
 
+  def self.cross_facility
```

Comment:
I lean toward `cross_facility`. `Facility.all_facility` would feel too close to `Facility.all` to me.